extensions/mssql/src/main.ts

```diff
@@ -297,6 +339,21 @@ async function handleOpenNotebookTask(profile: azdata.IConnectionProfile): Promi
 	}
 }
 
+async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionProfile, appContext: AppContext): Promise<void> {
+	const notebookRelativePath = 'notebooks/tsg/cluster-status.ipynb';
+	const notebookFullPath = path.join(appContext.extensionContext.extensionPath, notebookRelativePath);
+	if (!Utils.fileExists(notebookFullPath)) {
+		vscode.window.showErrorMessage(localize('fileNotFound', 'Unable to find the file specified.'));
```

Comment:
Minor: we're meant to use `""` for localized strings, e.g.
```suggestion
		vscode.window.showErrorMessage(localize('fileNotFound', "Unable to find the file specified"));
```
Also usually no need for `.` at the end in these messages.

Reply:
I just followed the other localize examples in the file. I'll update them, Thanks!